.gitignore

```diff
@@ -22,8 +22,7 @@ build/Release
 # Dependency directory
 # Commenting this out is preferred by some people, see
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git-
-/node_modules/
-/typings/
+node_modules
```

Comment:
By adding both typings and types in gitignore we don't support both, we just ignore both.